apps/server/src/modules/h5p-editor/contentStorage/contentStorage.spec.ts

```diff
@@ -174,22 +266,34 @@ describe('ContentStorage', () => {
 		});
 		describe('WHEN content can not be added error', () => {
 			it('should throw new error', async () => {
-				const { metadata, content, user } = setup();
-				jest.spyOn(fsPromiseMock, 'writeFile').mockImplementationOnce(() => {
+				const { metadata, content, user, contentId } = setup();
+				s3ClientAdapter.create.mockImplementationOnce(() => {
```

Comment:
It makes no sense to mock the s3ClientAdapter here because addContent does not interact with it. It only interacts with the repo.